Packs/CommonScripts/Scripts/KeyValueToJson/KeyValueToJson.py

```diff
@@ -0,0 +1,50 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+def main():
+    args = demisto.args()
+    values = args.get('value')
+    keys = args.get('key')
+
+    if not isinstance(values, list):
+        try:
+            values = json.loads(values)
+        except TypeError:
+            pass
+        if not isinstance(values, list):
+            try:
+                values = values.split(",")
+            except AttributeError:
+                pass
+
+    if not isinstance(values, list):
+        return_error("Canoot convert the input values into a list")
+
+    if not isinstance(keys, list):
+        try:
+            keys = json.loads(keys)
+        except TypeError:
+            pass
+        if not isinstance(keys, list):
+            try:
+                keys = keys.split(",")
+            except AttributeError:
+                pass
+
+    if not isinstance(keys, list):
+        return_error("Canoot convert the input key into a list")
```

Comment:
`argToList` basically does all that, so this is redundant

Reply:
Removed